tests/test_packages/test_connections/test_oef/test_communication.py

```diff
@@ -973,9 +971,7 @@ def _node(self):
 @pytest.mark.asyncio
 async def test_send_oef_message(network_node):
     """Test the send oef message."""
-    private_key_pem_path = os.path.join(CUR_PATH, "data", "priv.pem")
-    wallet = Wallet({FETCHAI: private_key_pem_path})
-    address = wallet.addresses[FETCHAI]
+    address = FetchAICrypto().address
```

Comment:
we now use a new address for every test, rather than the same address associated with the private key `test/data/priv.pem`. The same has been applied to other tests.

The reason is that, on the OEF side, using the same address for testing might cause some problems due to delays. This should not happen if all the tests clean up the resources correctly. However, we have better test independence with this approach.